README.md

```diff
@@ -188,17 +238,22 @@ You should see an error caused by the kube-system DNS service like this:
 To post the session into CAS, we will use the `scone-cli` container again.
 
 ```bash
+# Provide CLI address for next commands
+export SCONE_CLI_ADDRESS=$(kubectl -n scone get pods \
+  -o=jsonpath='{.items[0].metadata.name}' -l app=scone-cli)
+
+# Copy over Service 2 session
 kubectl cp services/2_spiffe_aware_service/session.yaml \
-    scone/$(kubectl get pods -n scone -o=jsonpath='{.items[0].metadata.name}' \
-    -l app=scone-cli):2_spiffe_aware_service-session.yaml
+    scone/$SCONE_CLI_ADDRESS:2_spiffe_aware_service-session.yaml
 
-kubectl exec -it -n scone $(kubectl get pods -n scone -o=jsonpath='{.items[0].metadata.name}' \
-    -l app=scone-cli)  -- /bin/bash
+# SSH into CLI container
+kubectl exec -it -n scone $SCONE_CLI_ADDRESS -- /bin/bash
 
 # Inside the scone-cli container
 HASH_SERVICE_2=$(scone session create 2_spiffe_aware_service-session.yaml --use-env)
+
 # Get the session hash for Service 2
-echo $HASH_SERVICE_2
+echo "HASH_SERVICE_2=$HASH_SERVICE_2"
```

Comment:
I think we can have a quick win here.

`echo "export HASH_SERVICE_2=$HASH_SERVICE_2"`

then ask the user to copy the entire line. It is a small change that gains the user's sympathy. The same can be done for service 3.